src/openforms/config/setup_configuration/models.py

```diff
@@ -0,0 +1,32 @@
+from django_setup_configuration.models import ConfigurationModel, DjangoModelRef
```

Comment:
I wasn't entirely sure on what directory to place these files in, should they be in `src/openforms/config/...` or in `src/openforms/contrib/...`?

Reply:
hrm, will have to have a think, I think it'll partly be both? e.g. 3rd party library stuff like zgw consumers in `config`, and openforms constructs (like objectsapigroup) in `contrib` close to where the models are defined

Reply:
discussed, I've moved this to `contrib/objects_api/setup_configuration`